tools/purge_ems.rb

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env ruby
+enviroment = File.expand_path('../config/environment', __dir__) + ".rb"
+require enviroment if File.exist?(enviroment)
+
+if !defined?(Vmdb::Application)
+  warn "This script must exist in the tools directory or can live elsewhere but needs to be run via rails runner path/to/purge_ems.rb --id 1234 --follow true"
+  exit 1
+end
+
+require 'optimist'
+
+ARGV.shift if ARGV[0] == '--' # if invoked with rails runner
+opts = Optimist.options do
+  banner "Purge managements system records in the background via the queue.\n\nUsage: ruby #{$0} [options]\n\nOptions:\n\t"
+  opt :id,      "Mangement System id",                                         :short => :i, :type => :integer,                    :required => true
+  opt :follow,  "Follow the progress or exit after creating messages",         :short => :f, :type => :string,  :default => "true"
```

Comment:
Prefer making this `:type => :boolean, :default => true`.  Then
1. there's no need for the validation below
2. The CLI would be either `--follow` or `--no-follow` with `--follow` being the default

Reply:
I can try it.  I was trying boolean with --follow true and --follow false and that was a cluster so I ditched it as less important to testing it. 

`--follow` and `--no-follow` should be fine.